optax/_src/alias.py

```diff
@@ -431,6 +431,29 @@ def noisy_sgd(
   )
 
 
+def online_newton_step(learning_rate: ScalarOrSchedule,
+                       eps: float) -> base.GradientTransformation:
+  # pylint: disable=line-too-long
+  """Online Newton Step optimizer.
+  (see the description of the ONS in Fig. 2 p. 176 of the reference below).
```

Comment:
According to the google python style guide there needs to be a blank line following the summary line.